code/style.css

```diff
@@ -1,58 +1,333 @@
+html {
+  scroll-behavior: smooth;
+}
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: 'Courier New', Courier, monospace;
+  box-sizing: border-box;
+  background-color: rgb(65, 65, 65);
 }
 
-p {
+h1, h2, h3 {
+  font-family: 'Ribeye Marrow', cursive;
+  font-size: 38px;
+  color: #fff;
+}
+
+h2 {
+  background: rgb(138, 22, 45);
+}
+
+h3 {
+  text-align: center;
+}
+
+.ingress {
   margin: 1px;
   font-weight: 300;
-  background-color: #fff;
+  color: #fff;
+  background-color: rgb(138, 22, 45);
+}
+
+p {
+  padding: 5px;
+  margin-top: 10px;
+  font-weight: 300;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+/* The hero image */
+.hero-image {
+  /* Use "linear-gradient" to add a darken background effect to the image (photographer.jpg). This will make the text easier to read */
+  width: 100%;
+  height: 100vh;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("img/mansion_hero.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  /* Position and center the image to scale nicely on all screens */
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+/* Place text in the middle of the image */
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);}
+  
+  /* LETTER STYLE */
+.container {
+  display: inline-block;
+  justify-content: center;
+  max-width: 80%;
+  margin-bottom: 20px;
+}
+
+.letter {
+  font-family: 'Courier New', Courier, monospace;
+  font-style: italic;
+  text-align: center;
+  margin: 50px auto;
+  height: fit-content;
+  padding: 10px 10px 20px 10px;
+}
+
+.letter p {
+  text-decoration: underline;
+}
+
+div.polaroid {
+  text-align: left;
+  width: 284px;
+  padding: 10px 10px 20px 10px;
+  border: 1px solid #BFBFBF;
+  background-color: rgb(255, 248, 221);
+  box-shadow: 10px 10px 5px #222222;
+}
+
+div.rotate_right {
+  width: 284px;
+  float: left;
+  -ms-transform: rotate(7deg); /* IE 9 */
+  -webkit-transform: rotate(7deg); /* Safari */
+  transform: rotate(7deg);
+}
+
+div.rotate_left {
+  width: 284px;
+  float: left;
+  -ms-transform: rotate(-8deg); /* IE 9 */
+  -webkit-transform: rotate(-8deg); /* Safari */
+  transform: rotate(-8deg);
+}
+
+.polaroidImg {
+  width: 284px;
+  height: 213px;
+}
+
+/* CARD STYLE */
+
+.cardShow, .mysteryShow{
+  display: none;
 }
 
 .cards {
+  display: flex;
   margin: 50px auto;
-  width: 90%;
+  width: 95%;
   height: 600px;
-  background: #ccc;
-  display: flex;
+  background: url(img/table.jpg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`url("img/table.jpg");`